README.md

```diff
@@ -60,6 +60,53 @@ The repository has a `.vscode` folder that contains `settings.json` and `extensi
 
 7. Change the .env.template to .env.local and all is done.
 
+## How to create a profile table in Supabase
```

Comment:
On line 59, we have:

> 6. Click User Management Starter and then run

Is this duplicated now?

Reply:
Yup, that template has a query for creating `profiles` table in it. I'll update the SQL code to only include the `trigger` and `function` creation queries. Can you check and let me know whether our current database also has this `profiles` table? If yes, you only need to run the following queries:

```SQL
create function public.handle_new_user()
returns trigger
language plpgsql
security definer set search_path = public
as $$
begin
  insert into public.profiles (id, username)
  values (new.id, new.raw_user_meta_data->>'username');
  return new;
end;
$$;

create trigger on_auth_user_created
  after insert on auth.users
  for each row execute procedure public.handle_new_user();
```

Reply:
Yes, following the current instructions in README, the current database has the profiles table.

Do you think we should remove that part of the instruction and include the full SQL in README?

Reply:
Yes, a full SQL will be good. I think we should create a file with all the SQL queries (that we will keep updating) and ask the reader to execute those at step 6. What do you think?

Reply:
That sounds like a good idea!